frontend/src/data/config/MetricConfigSDOH.ts

```diff
@@ -15,6 +15,7 @@ export const SDOH_CATEGORY_DROPDOWNIDS = [
   'avoided_care',
   'health_insurance',
   ...(SHOW_GUN_VIOLENCE ? ['gun_violence'] : []),
+  'gun_violence_youth',
```

Comment:
let's move youth into that array, and treat the entire thing as being behind a flag